om4labs/diags/acc_drake/acc_timeseries.py

```diff
@@ -0,0 +1,161 @@
+def parse():
+    return
+
+
+def read(dictArgs):
+    dset = xr.open_mfdataset(dictArgs["infile"])
+    return dset
+
+
+def ACC_Transport(darray, zdim="z_l", ydim="yh"):
+    """Calculate ACC Transport by summing across dimensions
+
+    Parameters
+    ----------
+    darray : xarray.DataArray
+        Data Array containing transport
+    zdim : str, optional
+        Name of vertical dimension, by default "z_l"
+    ydim : str, optional
+        Name of latitude dimension, by default "yh"
+
+    Returns
+    -------
+    xarray.DataArray
+        Data Array containing time series of transport
+    """
+    darray = darray.sum(dim=(zdim, ydim))
+    darray = darray * (1 / 1035) * 1.0e-6
```

Comment:
```suggestion
    darray = darray * (1. / 1035.) * 1.0e-6
```

Consider making this floating point